WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -57,8 +58,8 @@
             = "KEY_IS_SHOWING_REMOVE_PLUGIN_CONFIRMATION_DIALOG";
 
     private SiteModel mSite;
-    private PluginModel mPlugin;
-    private PluginInfoModel mPluginInfo;
+    private SitePluginModel mPlugin;
```

Comment:
Should we rename this to `mSitePlugin`?

Reply:
Done in d142e556ea08e68826ac3c68dac75aeb2c499061. I also updated a couple comments to say wp org plugin instead of plugin information.

P.S: I forced pushed some changes to avoid an extra commit just for a comment change.